app/build.gradle

```diff
@@ -24,6 +24,8 @@ android {
         main.res.srcDirs += 'src/main/lib/android-ripple-background/library/src/main/res/'
         main.java.srcDirs += 'src/main/lib/material-intro-screen/material-intro-screen/src/main/java/'
         main.res.srcDirs += 'src/main/lib/material-intro-screen/material-intro-screen/src/main/res/'
+        main.java.srcDirs += 'src/main/lib/powerampapi/poweramp_api_lib/src/'
```

Comment:
Is there really no other way to import that lib ? (maybe from a maven repository or something, I don't know)
I would rather only use that kind of import as a last resort. It's always a pain to maintain

Reply:
I wasn't seeing a maven/gradle option right away so I just went with the way you used before and called it a day. Will do some more research if it's possible via dependencies.